tests/Integration/Database/EloquentWithCountTest.php

```diff
@@ -112,7 +131,7 @@ protected static function boot()
         parent::boot();
 
         static::addGlobalScope('app', function ($builder) {
-            $builder->where('idz', '>', 0);
+            $builder->where('id', '>', 0);
```

Comment:
Why did you modified this test?

Reply:
The model / relationship is not actually used in the tests, and could probably be removed entirely, but the global scope constraint seems to be a typo. Do correct me if I'm wrong.

Reply:
I'd revert this. It's in general best to keep changes limited only related to the PR at hand.

Reply:
@driesvints I was thinking that as I made the commit, but I'll send in a separate PR. If you could verify that the third model isn't needed here I'll clean up the entire test in one go.